rust/kcl-lib/src/lsp/tests.rs

```diff
@@ -16,6 +16,12 @@ use crate::{
     parsing::ast::types::{Node, Program},
 };
 
+// TODO: Revert this before merging
+#[tokio::test(flavor = "multi_thread", worker_threads = 12)]
+async fn test_example_failing_test_to_disable() {
+    assert_eq!(1, 2);
+}
+
```

Comment:
See the history of this new test failing, as expected, and then being disabled via TAB: https://test-analysis-bot.hawk-dinosaur.ts.net/projects/KittyCAD/modeling-app/results?search=example_failing_test&branch=dynamic-skip-junit&show=all